projects/platform/src/lib/pipes/log.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'log'
+})
+export class LogPipe implements PipeTransform {
+
+  transform(value: any, ...args: any[]): any {
```

Comment:
Can't you just drop args in this case?

Reply:
![image](https://user-images.githubusercontent.com/13664201/55412423-15af0300-5570-11e9-97a5-e18183a04900.png)
I do not know.

Reply:
`console.log` accepts and prints any number of arguments by the spec: https://console.spec.whatwg.org/#logger
But in this case your pipe doesn't really have any additional arguments.
You could use them to provide log level though.
For example:
```html
{{ value | log:'error' }}
```

```ts
transform(value: any, logLevel: 'log' | 'warn' | 'error' = 'log'): any {
  console[logLevel](value)
  return value
}
```


Technically, your current implementation can be used in a such manner:
```html
{{ '%d two %s' | log:[1, 'three'] }}
```
which will result in:
![image](https://user-images.githubusercontent.com/20531968/55416278-2adb6000-5577-11e9-88da-5cddef958901.png)

But it's hard to image a use case (which could be good in terms of ngxf lol).

Reply:
I think we need just implements others pipes: InfoPipe, WarnPipe and ErrorPipe 👌

Reply:
And what about console.profile, console.time and console.group?

Reply:
> А как насчет console.profile, console.time и console.group?

I will do it if you see it fit.

Reply:
> А как насчет console.profile, console.time и console.group?

I will do it if you see it fit.

Reply:
@oOlDeMoNlOo already implemented)
https://github.com/ngxf/platform/commit/1ad2fa971545d800ad612949ad02959754c9c48c